client/core/simnet_trade.go

```diff
@@ -1985,7 +1977,7 @@ func (s *simulationTest) assertBalanceChanges(client *simulationClient) error {
 	}
 
 	var baseFees, quoteFees int64
-	if fees := ord.FeesPaid; fees != nil {
+	if fees := ord.FeesPaid; fees != nil && !isRefund {
```

Comment:
This caused account tests to fail and was incorrect for utxo refunds as well, but the amount was not enough to matter for them.

Reply:
If it's refunded, `fees.Swap` was paid though (plus refund fees for the same asset), just not `fees.Redemption`.

Reply:
The expected balances are checked after the swap and then after the refunds, so `fees.Swap` was already accounted for. I think.